### ISSUE TITLE
Improperly encoded/decoded filename 

### ISSUE THREAD

**Issue by SidIcarus** [liximomo/vscode-sftp#285]
## Expected vs Actual Behavior
File upload - Error `URI malformed`

## Steps to Reproduce the Problem

1. create a file and name it with unescaped characters
    - example: `/path/to/%etc%profile.d%.sh_function`
2. edit file
3. upload to remote

### Extension Logs

  ```
  [trace] execute upload for /path/to/%etc%profile.d%.sh_function
  [error] URIError: URI malformed
    at decodeURIComponent (<anonymous>)
    at Function.e.parse (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:47:473)
    at Function.makeResource (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/core/UResource.js:60:43)
    at Function.from (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/core/UResource.js:83:39)
    at FileCommand.<anonymous> (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:65:50)
    at Generator.next (<anonymous>)
    at /path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:7:71
    at Promise (<anonymous>)
    at __awaiter (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:3:12)
    at FileCommand.handleFile (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:50:16)
    at targetList.map.t (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:40:59)
    at Array.map (native)
    at FileCommand.<anonymous> (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:40:45)
    at Generator.next (<anonymous>)
    at fulfilled (/path/to/.vscode/extensions/liximomo.sftp-1.3.7/out/src/commands/FileCommand.js:4:58)
    at <anonymous>
  ```

## Specifications

  - Version: 1.37
  - Platform: MacOS 10.13.6 (17G65)

**Comment by liximomo** [comment]
Fix in v1.3.8

**Comment by SidIcarus** [comment]
Thanks @liximomo !
